renderlib/SceneView.h

```diff
@@ -58,5 +59,8 @@ struct SceneView
   Scene* scene = nullptr;
   RenderSettings* renderSettings = nullptr;
 
-  bool anythingActive() const { return scene != nullptr; }
+  // weak ptr! must not outlive the objects it points to.
+  SceneObject* selection = nullptr;
```

Comment:
key change: this selection object will be able to be affected by transform manipulators